screens/IncidentList.js

```diff
@@ -202,30 +202,66 @@ class IncidentList extends React.Component {
     return (
       <IncidentCard
         data={incident}
-        onPress={() =>
-          this.props.navigation.navigate('IncidentDetail', {
-            incidentDetail: incident,
-          })
-        }
+        onPress={() => {
+          if (incident.id === 0) {
+            this.props.navigation.navigate('IncidentDetailSample')
+          } else {
+            this.props.navigation.navigate('IncidentDetail', {
+              incidentDetail: incident,
+            })
+          }
+        }}
       />
     )
   }
 
   renderCarousel() {
-    if (this.props.incidents.length === 0) {
-      return (
-        <View style={styles.emptyIncidentBox} pointerEvents={'none'}>
-          <Text style={{ fontSize: 13, color: '#4a4a4a' }}>
-            {i18n.t('empty_incidents')}
-          </Text>
-        </View>
-      )
-    }
+    const SampleIncident = [{
+      Progresses: [{
+        IncidentId: 2,
+        content: "상황 종료되었습니다. 정확한 사고 경위를 정리하여 빠른 시일 내에 발표하겠습니다.",
+        createdAt: "2018-09-03T13:32:21.763Z",
+        id: 1,
+        updatedAt: "2018-09-03T13:32:21.763Z",
+      }],
+      User: {
+        createdAt: "2019-09-03T13:14:38.173Z",
+        displayname: "Jeong, Jinwoo",
+        id: 1,
+        isAdmin: false,
+        kaist_uid: "00088348",
+        ku_kaist_org_id: "4421",
+        ku_kname: "정진우",
+        mobile: "010-4602-9500",
+        updatedAt: "2019-09-03T13:19:25.057Z",
+      },
+      UserId: 1,
+      contract: null,
+      createdAt: "2019-09-01T08:58",
+      id: 0,
+      lat: "36.374356",
+      lng: "127.3605294",
+      state: "완료",
+      type: "샘플",
+      updatedAt: "2019-09-01T08:58",
+    }]
+    const incidentsWithSample = this.props.incidents.length == 0 ? SampleIncident : this.props.incidents.concat(SampleIncident)
+    
+    // if (this.props.incidents.length === 0) {
+    //   return (
+    //     <View style={styles.emptyIncidentBox} pointerEvents={'none'}>
+    //       <Text style={{ fontSize: 13, color: '#4a4a4a' }}>
+    //         {i18n.t('empty_incidents')}
+    //       </Text>
+    //     </View>
+    //   )
+    // }
+
     return (
       <View style={styles.carouselContainer}>
         <Pagination
-          dotsLength={this.props.incidents.length}
-          activeDotIndex={this.props.indexSelected}
+          dotsLength={incidentsWithSample.length}
```

Comment:
where did the incidents from the server go?
we need array.concat or something

Reply:
There is this.props.incidents.concat on line 248.

Reply:
oh I see